packages/x-date-pickers-pro/src/MobileDateRangePicker/MobileDateRangePicker.types.ts

```diff
@@ -28,6 +28,7 @@ export interface MobileDateRangePickerProps<
   TDate extends PickerValidDate,
   TEnableAccessibleFieldDOMStructure extends boolean = false,
 > extends BaseDateRangePickerProps<TDate>,
+    MobileOnlyPickerProps,
```

Comment:
We are extending with a bigger than expected type, which destroys very relevant comments on `inputRef`, `label`, and `name` props.
Which type could we extend to avoid the overspecification? 🤔